src/jit/block.cpp

```diff
@@ -1375,15 +1375,15 @@ BasicBlock* Compiler::bbNewBasicBlock(BBjumpKinds jumpKind)
 }
 
 //------------------------------------------------------------------------
-// blockHasEHFlowIn: Determine if this block has incoming exception flow.
+// blockhasEHBoundaryIn: Determine if this block has an incoming EH boundary.
```

Comment:
What is an incoming boundary? I understand what an incoming edge means, but it is hard for me to understand what is an incoming boundary.

Reply:
Would "this block begins at an EH boundary" be clearer?